source/server-selection/tests/server_selection/ReplicaSetWithPrimary/read/Secondary.yml

```diff
@@ -3,18 +3,18 @@ topology_description:
   servers:
   - &1
     address: b:27017
-    avg_rtt_ms: 5
+    avg_rtt_ms: 21 # outside the latency window if primary is considered
     type: RSSecondary
     tags:
       data_center: nyc
   - &2
     address: c:27017
-    avg_rtt_ms: 100
+    avg_rtt_ms: 100 # outside the latency window if both secondaries are considered
```

Comment:
I don't understand the comment.  This server is outside the latency window regardless of whether the primary is considered.  Consider having three secondaries, where the second is in the latency window, and the third is outside.

Reply:
That's right, it's the same as the test used to be: this server will _always_ be outside the latency window, and so never show up in the `in_latency_window` list. Maybe it would be more clear if I removed the comments? I'm not sure that adding another secondary gives us more coverage

Reply:
Yeah, the comment is definitely confusing and should be changed.

Regarding the test coverage: I reviewed all of the existing tests and found that there are no tests that have more than one server within the latency window.  Given the semantics of the latency window (which is all about which servers that have an RTT higher than the fastest suitable server should be selected), that seems like an oversight that should be corrected.

Reply:
To close the loop on this.

1. Please update the comment
2. You asked whether adding another secondary gives us more coverage.  I think the answer to that is yes, since we have no existing tests that assert that any more than the fastest server is considered in the latency window.   But even if that's the case, if you consider this additional coverage as out of scope for what you're trying to accomplish here, then I'll accept that and LGTM.